classes/Zipcode.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/**
+ * Value type wrapping validation logic for zipcode.
+ */
+class Zipcode {
+    public static $ZipcodeValidationRegex = '/^[0-9]{4}[a-zA-Z]{2}$/';
```

Comment:
- please start variable names with lower case letter
- Why not make this a `private const` as that's probably a bit better

Reply:
How do you then tell the difference between a private and public class field if both are lower camel cases?

Reply:
Just from reading the code, you don't. What you'll usually do is make getter/setter methods from them so you can't access it directly and I don't see the need here either. 

And, as I suggested to make it a `private const` you should fully upercase with `_` so it will become: `private const ZIP_VALIDATION_REGEX` to only be used internally.

Reply:
I do agree a constant is better here.